src/server/table.h

```diff
@@ -142,4 +140,11 @@ struct DbTable : boost::intrusive_ref_counter<DbTable, boost::thread_unsafe_coun
 // the snapshot process. We copy the pointers in StartSnapshotInShard function.
 using DbTableArray = std::vector<boost::intrusive_ptr<DbTable>>;
 
+// Before acquiring and releasing keys, one must "normalize" them via GetLockTag().
+LockTag GetLockTag(std::string_view key);
+
+inline LockFp GetLockFp(std::string_view key) {
+  return LockTable::Fingerprint(GetLockTag(key));
+}
```

Comment:
And if we have a good type, we can chain LockTag{key}.Fingerprint() or convert implicitly

Reply:
sure.